apps/src/applab/ExternalRedirectDialog.jsx

```diff
@@ -15,6 +15,8 @@ const styles = {
   }
 };
 
+const DIALOG_WIDTH = 700;
```

Comment:
Where did this const come from? (might be worth adding a comment and/or moving it to the src/applab/constants file.